README.md

```diff
@@ -1,36 +1,52 @@
-# Friday Night Funkin
+# Friday Night Funkin!
 
-This is the repository for Friday Night Funkin, a game originally made for Ludum Dare 47 "Stuck In a Loop".
+![FNF_Logo](https://user-images.githubusercontent.com/72235309/112068307-08fe8d80-8b40-11eb-84d0-fc64a10dd845.png "FNF Logo")

```

Comment:
I'd use a transparent logo; it would look pretty amongst the other transparent images used.

It would also be great to place the images used in the 'art' folder, just in case they get removed by Github.

Reply:
Alright I'll find one and use it!